src/convenience/frameworks.node.ts

```diff
@@ -53,11 +53,10 @@ const awsLambda = (event: any, _context: any, callback: any) => ({
 const azure = (context: any, req: any) => ({
     update: Promise.resolve(req.body),
     header: context.res.headers[SECRET_HEADER],
-    end: () =>
-        context.res = {
-            status: 200,
-            body: "",
-        },
+    end: () => (context.res = {
```

Comment:
This change is useless and only makes the code harder to read, please revert

Reply:
It was generated after running `deno fmt` but sure.

Reply:
Oh

Reply:
So what should i do? It still formats after i save the file

Reply:
Then we can perform the changes that way. I'll try to reproduce it and then open a bug report for the Deno formatter. Don't worry about it for now.

Reply:
Okay, thanks!